modules/ztp-using-pgt-to-configure-performance-mode.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-using-pgt-to-configure-performance-mode_{context}"]
+= Configuring performance mode using PolicyGenTemplate CRs
+
+`PolicyGenTemplate` custom resources (CRs) allow you to overlay additional configuration details onto the base source CRs provided with the GitOps plugin in the `ztp-site-generate` container.
+
+Follow this example to set performance mode by updating the `workloadHints` fields in the generated `PerformanceProfile` CR for the reference configuration, based on the `PolicyGenTemplate` CR in the `group-du-sno-ranGen.yaml`.
```

Comment:
These lines are repeated 4 times throughout the documentation. Please re-write the other sections with a general procedure for updating the source-crs and leave the sub-sections to contain just the workloadHints specification for the various power modes.